habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -180,7 +184,10 @@ def _collect_rollout_step(
 
         t_step_env = time.time()
 
-        outputs = self.envs.step([a[0].item() for a in actions])
+        step_data = [a[0].item() for a in actions]
+        profiling_utils.range_pop()  # compute actions
```

Comment:
I want my range to end after the call to `item()` but before the call to `envs.step`. I'm relying on `item()` as a blocking/synchronizing call for the preceding GPU inference for computing actions.